### DISCUSSION TITLE
Phoenix 1.6 defaults the test endpoint's server to `false`

### DISCUSSION THREAD

**Discussion by sodapopcan** [keathley/wallaby#627] [Q&A] [answered]
In Phoenix 1.6-rc.0, the `server` option in the endpoint config is now `false` which causes Wallaby not to work.  Of course, it's still a release candidate so this may or not stick around but I thought I would report.

Here is the config in question in `config/test.exs`:

```elixir
config :my_app, MyAppWeb.Endpoint,
  http: [ip: {127, 0, 0, 1}, port: 4002],
  server: false
```

**Comment by mhanberg** [comment, marked as answer]
That has always been false I believe. In the Wallaby docs, you are instructed to set that to true, to start the web server.

**Reply by sodapopcan** [reply]
Oh, ok, I see what I did.  I'm still pretty new to setting up Phoenix apps and not used to the config.  The docs don't mention the generated `http` key so I didn't recognize it as always being set and double-set the endpoint config.
